test/functional/wallet_groups.py

```diff
@@ -63,5 +65,29 @@ def run_test (self):
         assert_approx(v[0], 0.2)
         assert_approx(v[1], 1.3, 0.0001)
 
+        # Empty out node2's wallet
+        self.nodes[2].sendtoaddress(address=self.nodes[0].getnewaddress(), amount=self.nodes[2].getbalance(), subtractfeefromamount=True)
+        self.sync_all()
+        self.nodes[0].generate(1)
+
+        # Fill node2's wallet with 10000 outputs corresponding to the same
+        # scriptPubKey
+        for i in range(5):
+            raw_tx = self.nodes[0].createrawtransaction([{"txid":"0"*64, "vout":0}], [{addr2[0]: 0.05}])
```

Comment:
Side note, I wonder if `createrawtransaction` should allow duplicate addresses now that outputs is an array? cc @MarcoFalke 

BTW, #12419 would break this — maybe I should close that?